spec/helpers/sticky_footer_visibility_spec.rb

```diff
@@ -0,0 +1,92 @@
+RSpec.describe StickyNewsletterVisibility, type: :helper do
```

Comment:
Other thing that I realize. Can you rename this file to `sticky_newsletter_visibility_spec.rb`?